tests/test_csc.py

```diff
@@ -141,6 +142,29 @@ def setUp(self):
             # immediately).
             self.patch.return_value.AVS_Done.assert_called()
 
+    async def test_connect_by_index(self):
+        """Test that changing the index number changes the serial number
+        that we attempt to connect to.
+        """
+        # Mock two connected devices (we will connect to the second).
+        n_devices = 2
+        serial_number = "1606191U1"  # this corresponds to ATBroad (index=3)
```

Comment:
Why not use the mapping you already made that contains these values? It seems like it might be a useful thing to expose as a public constant.